pypdf/_writer.py

```diff
@@ -3300,22 +3300,20 @@ def _set_page_label(
         self._root_object[NameObject(CatalogDictionary.PAGE_LABELS)] = page_labels
 
 
-def _pdf_objectify(obj: Union[Dict[str, Any], str, int, List[Any]]) -> PdfObject:
+def _pdf_objectify(obj: Union[Dict[str, Any], str, float, List[Any]]) -> PdfObject:
```

Comment:
While it might be allowed, I prefer explicit over implicit.

Reply:
Same. I tried to have both int and float but it causes an error, and PEP 484 means float is preferred.

Reply:
Where did you see this issue?

Reply:
Tried before.